### ISSUE TITLE
Place under the menu bar

### ISSUE THREAD

**Issue by msaspence** [mattDavo/Yippy#10]
Yippy gets layered over the menu bar's drop shadow/border, which makes it look a little out of place at the top. This is true is dark mode but much more evident in light more imo. 

<img width="424" alt="image" src="https://user-images.githubusercontent.com/242861/87144474-b45ffe80-c29f-11ea-96ae-1d40927e5c51.png">

PS This is a great little app, I've spent so long looking for a clipboard manager that suits me. (The only thing significant thing missing for me right now is search). So thanks.

**Comment by mattDavo** [comment]
Hi @msaspence, thank you for your support! Yes search seems to be the top priority (for me as well) but I really cannot commit to it right now, and is quite tricky with the way Yippy avoids taking window focus.

Hmm yes I understand your concern, but unfortunately this is the best solution that I have found. ☹️  Apple does not provide a great API for determining the visible frame (screen minus dock minus menu bar) at all times. And this is the best compromise that does not make it look weird when the dock or menu bar is hidden (for example in full screen apps). Apple actually has this problem themselves! I copied their solution for this problem with their notifications menu 😜 

<img width="413" alt="Screen Shot 2020-07-24 at 9 53 48 pm" src="https://user-images.githubusercontent.com/13739650/88388593-297f1800-cdf8-11ea-9f31-0f11774b0690.png">
